packages/cardano-services/test/StakePool/DbSyncStakePoolProvider/__snapshots__/StakePoolBuilder.test.ts.snap

```diff
@@ -1355,23 +1363,29 @@ exports[`StakePoolBuilder queryPoolMetrics sort by default sort 1`] = `
 Array [
   Object {
     "activeStake": 0n,
+    "activeStakePercentage": 0,
     "blocksCreated": "0",
     "delegators": "1",
     "livePledge": 999999828559n,
+    "liveStake": 999999828559n,
     "saturation": 0.011898674769756331,
   },
   Object {
     "activeStake": 2986376991n,
+    "activeStakePercentage": 6.02744522378192,
```

Comment:
🤔

Reply:
There is something off here, `active stake` is higher than `live stake`, but this shouldn't be possible since now `live stake` contains `active stake`.

Reply:
Seems like #467 addresses the stake issue?